spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -0,0 +1,119 @@
+<!-- omit in toc -->
+# CCV: Overview and Basic Concepts
+[&uparrow; Back to main document](./README.md)
+
+<!-- omit in toc -->
+## Outline
+- [Security Model](#security-model)
+- [Motivation](#motivation)
+- [Definitions and Overview](#definitions-and-overview)
+  - [Channel Initialization](#channel-initialization)
+  - [Validator Set Update](#validator-set-update)
+
+
+
+# Security Model
```

Comment:
I made some changes that I think make this section easier to read. 
- I moved the entire single chain behavior in a Background paragraph. If we add eventually a Background chapter (which we probably should), then I'll move all these Background paragraphs there. I'd not just remove them since they contain important info especially for readers that do not have a deep understanding of Cosmos. 
- I left the CCV implications on the security model (e.g., how is the unbonding period affected). I find it important to understand the rest of the specification.
- Regarding the assumptions made by CCV, these are described in the [Assumption section](https://github.com/cosmos/ibc/blob/marius/ccv-init-and-vsc/spec/app/ics-028-cross-chain-validation/system_model_and_properties.md#assumptions).